docs/stack/use-cases/_index.md

```diff
@@ -0,0 +1,417 @@
+
+When you model a supply chain with a graph, you can get interesting insights out of your data, helping you make important business and logistic decisions. For example, a graph database can help you recalculate an alternative route from factory A to consumer B during issues by answering these questions:
+
+- Where did this food contamination start and where it could have spread?
+- What is going to be the downstream consequence of this manufacturer reporting that they couldn't obtain enough copper? Which parts are they not going to be able to produce, and how will that influence our distribution network and retailers?
+- What is the spread of my product (in which countries/cities is it used)?
+- In which factories/storages has this product been before arriving in a specific shop?
+- What is the fastest way to move from factory A to consumer B?
+- What is the most cost-effective way to move from factory A to consumer B?
+
+**Recommendation engine (social networks)**
+
+A graph database can help you highlight for user A a post that many of their friends have commented on and recommend videos with a global score of above 4 by answering these questions:
+
+- Who are the friends of my friend?
+- Who are the friends of my friend A, who are also friends with person B?
+
+**Recommendation engine (retail)**
+
+A graph database can help recommend to user A products that their friends have bought, and that currently have a margin higher than X percent and large stock in a storage close to them by answering these questions:
+
+- What are other products in a similar price range and with a similar textile as what this user usually buys?
+- What are some products that other users have bought together with the product this user just added to their cart?
+- In which store in my city can I locate this item?
+
+**Geo navigation (commerce and travel)**
+
+- List all coffee shops in an area of 2 km, with an average rating higher than 3.5.
+- List all parks around me that my friends have been to.
+
+**Operations management (networks and IT)**
+
+- Which parts of the network does this customer rely on?
+- Which customers/applications will be affected if this network element fails?
+- Is there redundancy in the network for this customer?
+
+**Fraud detection**
+
+- Fraud in health care: Are there any doctors/clinics that are prescribing an unusually high number of a certain drug?
+
+- Fraud in telecom: Does this call/SMS follow a regular calling pattern? It is a new account that just dialed an unusually high number of people, with a short average call-time and was rejected many times, so the receiver of the call should be warned by a real-time notification.
+
+- Fraud in finance: Is this transaction between sender A and recipient B valid? Has the card used by the sender been reported as stolen? Has the device that the recipient is using ever been connected to a fraudulent account?
+
+### Learn more
+
+[RedisGraph](/docs/stack/graph)
+
+## RedisTimeSeries
+
```

Comment:
I think that none the examples below mention how multiple time series can be used.
Multiple airplanes, multiple stocks, etc.

Also, typical queries are not discussed. RedisTimeSeries is not just about storing data, but also about answering questions. What are these questions for each of these use cases?

Reply:
I agree. Happy to work with you on that.

Reply:
Would you like to make some suggestions? See my example for environmental measurements above.